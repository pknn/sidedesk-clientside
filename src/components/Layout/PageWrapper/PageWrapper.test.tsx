import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { PageWrapper } from '.'

describe('<PageWrapper />', () => {
  const mockChild: JSX.Element = <div id="mock-child">Hello</div>
  it('should render child component', () => {
    const shallowMountedComponent = shallow(
      <PageWrapper>{mockChild}</PageWrapper>,
    )

    expect(shallowMountedComponent.find('#mock-child').exists()).toBe(true)
  })
  it('should render correctly', () => {
    const shallowMountedComponent = shallow(
      <PageWrapper>{mockChild}</PageWrapper>,
    )

    expect(toJson(shallowMountedComponent)).toMatchSnapshot()
  })
})
