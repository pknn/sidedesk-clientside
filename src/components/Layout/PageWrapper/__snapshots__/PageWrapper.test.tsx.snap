// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageWrapper /> should render correctly 1`] = `
<div
  className="bg-gray-200 relative h-full min-h-screen"
>
  <div
    id="mock-child"
  >
    Hello
  </div>
</div>
`;
